supersetapiclient/roles.py

```diff
@@ -0,0 +1,15 @@
+"""Roles."""
+from dataclasses import dataclass
+
+from supersetapiclient.base import (
+    Object
+)
+
+
+@dataclass
+class Role(Object):
+    JSON_FIELDS = []
+    EXPORTABLE = True
```

Comment:
Are roles really exportable?

Reply:
No it isn't, at least there is no option in the user interface.

```suggestion
```